includes/fbproduct.php

```diff
@@ -226,15 +226,23 @@ public function get_all_image_urls() {
 
 		$image_urls = array();
 
-		$product_image_url        = wp_get_attachment_url( $this->woo_product->get_image_id() );
+		/**
+		 * Filters the FB product image size.
+		 *
+		 * @since x.x.x
+		 *
+		 * @param string $size The image size. e.g. 'full', 'medium', 'thumbnail'.
+		 */
+		$image_size 			  = apply_filters( 'facebook_for_woocommerce_fb_product_image_size', 'full' );
```

Comment:
Indent looks off.

Reply:
I can never remember if we do tabs or spaces :-D. I fixed it 👍🏾